include/view.h

```diff
@@ -30,6 +30,12 @@ enum ssd_preference {
 	LAB_SSD_PREF_SERVER,
 };
 
+enum ssd_mode {
+	LAB_SSD_MODE_NONE = false,
+	LAB_SSD_MODE_FULL = true,
+	LAB_SSD_MODE_BORDER,
```

Comment:
This is more readable if it progresses from least to most decorated, and doesn't abuse Booleans for their integer values. Setting `_NONE = 0` is sufficient to allow things like `if (!mode)` without over-specifying the other don't-care values.

```suggestion
	LAB_SSD_MODE_NONE = 0,
	LAB_SSD_MODE_BORDER,
	LAB_SSD_MODE_FULL,
```

Reply:
Unfortunately it's not that simple. I used this mapping so that existing calls to `view_set_decorations()` continue to work. I will have to change them, too. Maybe I will have time for that tomorrow.

Reply:
`false` should be the same as `0` and `true` should be the same as `1` so keeping the order but just removing `= true` and replacing `= false` with `= 0` should keep working. It is a bit strange though to upcast `bool` in such a way and IMHO we should update the callers as well.